src/__tests__/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].test.tsx

```diff
@@ -90,18 +91,61 @@ describe("pages/teachers/lessons", () => {
     expect(screen.queryByTestId("download-all-button")).not.toBeInTheDocument();
   });
 
-  it("share button is disabled with non legacy content", async () => {
-    render(
+  it("share button is not disabled with non legacy content", () => {
```

Comment:
The two tests have the same description, one passes null and the other `LEGACY_CONTENT` as lessonCohort, could potentially state the difference in them